src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -2,22 +2,28 @@
 
 import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.log4j.LogManager;
+import org.apache.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
+
     @Override
     public User login(String login, String password) throws AuthenticationException {
-        //TODO: add corresponding log message about method login was called
+        logger.info(String.format("Login method called with login: %s", login));
         User user = findByLogin(login);
         if (!user.getPassword().equals(password)) {
+            logger.error("Authentication failed: Username or password are incorrect");
             throw new AuthenticationException("Username or password are incorrect");
         }
+        logger.info("User successfully logged in with userId: " + user.getUserId());
```

Comment:
Similar to the previous log message, include parameter names in the log output. For example: logger.info("User successfully logged in with userId: {}", user.getUserId());